### PR TITLE
Update duckduckgo.po

### PR DESCRIPTION
**Pull request by ViktorOn**
Changes:
- Updated and fixed translation

### CODE REVIEW

**Comment by ViktorOn** [comment]
And I also want to ask a question about this:

изображение

Why did the translation "not reach" here?

File: locales/ru_RU/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -7453,7 +7453,7 @@ msgid ""
 "Viewing ads is privacy protected by DuckDuckGo. Ad clicks are managed by "
 "Microsoft's ad network"
 msgstr "DuckDuckGo обеспечивает защиту конфиденциальности пользователей при просмотре рекламных объявлений. Управление переходами по рекламным объявлениям осуществляется"
-" рекламной сетью «Майкрософт»"
+" рекламной сетью Майкрософт"
```
Comment (GioSensation) [inline locales/ru_RU/LC_MESSAGES/duckduckgo.po:7456]: In order to merge this, I need you to please also update the corresponding string in https://github.com/ViktorOn/duckduckgo-locales/blob/c1dafd342036217049ece8ca14e925b1ab3af012/t/internal_translations.t#L127. Thank you!

Comment (ViktorOn) [reply locales/ru_RU/LC_MESSAGES/duckduckgo.po:7456]: It's done

**Comment by GioSensation** [comment]
Merging now. There's a failing test, but I'll follow up right away. Thanks!

**Pull request merged**